src/cpp/database/database.cpp

```diff
@@ -5303,8 +5346,9 @@ Info Database::get_info(
             break;
         }
     }
-
+    
     return info;
+
 }
```

Comment:
Remove extra line break
```suggestion
    
    return info;
}
```